src/consoleReporter.js

```diff
@@ -8,6 +8,11 @@ const GREEN = '\x1b[32m';
 const RED = '\x1b[31m';
 const NORMAL = '\x1b[0m';
 
+let GioUnix = null;
+try {
+    GioUnix = (await import('gi://GioUnix')).default;
```

Comment:
This adds a top-level await. I think that's OK in terms of performance (nobody cares about startup time of jasmine-gjs and a GI import won't do disk I/O anyway assuming the library is already loaded) but it does bump the GJS version requirement to 1.71.1. So could you edit the .deb and .rpm packaging files to reflect that?

Alternatively, we could just fall back to the legacy importer for this conditional import. i.e. `GioUnix = imports.gi.GioUnix;`

Reply:
OK, done the former now.